dev/com.ibm.ws.concurrent.persistent_fat_auto_pollinterval/fat/src/com/ibm/ws/concurrent/persistent/fat/autonomicalpolling1serv/AutonomicalPolling1ServerTest.java

```diff
@@ -82,6 +83,8 @@ public static void tearDown() throws Exception {
     // This later results in a XA_RBTIMEOUT (106) error on the transaction (javax.transaction.xa.XAException).
     // We recover/retry just fine, but this is a reason why Derby is not used for production.
     @AllowedFFDC({
+        "jakarta.persistence.PersistenceException", // errors reported through Jakarta Persistence
+        "javax.persistence.PersistenceException", // errors reported through JPA
```

Comment:
Thanks for figuring this out!

In messages.log, I see two FFDCs being created:
```
[4/17/22, 18:44:51:061 PDT] 00000051 com.ibm.ws.logging.internal.impl.IncidentImpl                I FFDC1015I: An FFDC Incident has been created: "javax.transaction.RollbackException: Transaction rolled back com.ibm.tx.jta.TransactionImpl.registerSynchronization 2289" at ffdc_22.04.17_18.44.49.0.log
[4/17/22, 18:44:51:529 PDT] 00000051 com.ibm.ws.logging.internal.impl.IncidentImpl                I FFDC1015I: An FFDC Incident has been created: "javax.persistence.PersistenceException: 
```
So do we also need to add:
```
jakarta.transaction.RollbackException
javax.transaction.RollbackException
```
to all the `@AllowedFFDC` annotation?

Reply:
Good idea. It would be useful to add the jakarta version of RollbackException for if these tests run against EE9+ features